test/test_metalearning/pyMetaLearn/test_meta_features.py

```diff
@@ -16,475 +19,598 @@
+        meta_features.metafeatures["KurtosisSTD"](
+            X_transformed, y, logging.getLogger('Meta'), categorical_transformed)
 
-    def test_class_entropy_multilabel(self):
-        X, y = self.get_multilabel()
-        mf = self.mf["ClassEntropy"](X, y, self.logger)
+    def test_skewnesses(self, meta_train_data_transformed):
+        X_transformed, y, categorical_transformed = meta_train_data_transformed
+        mf = meta_features.helper_functions["Skewnesses"](
+            X_transformed, y, logging.getLogger('Meta'), categorical_transformed)
+        assert 6 == len(mf.value)
 
-        classes = [(16, 84), (8, 92), (68, 32), (15, 85), (28, 72)]
-        entropies = []
-        for cls in classes:
-            cls = np.array(cls, dtype=np.float32)
-            cls = cls / sum(cls)
-            entropy = -np.sum([c * np.log2(c) for c in cls])
-            entropies.append(entropy)
-
-        self.assertAlmostEqual(mf.value, np.mean(entropies))
-
-    def test_landmark_lda(self):
+    def test_skewness_min(self, meta_train_data_transformed):
+        X_transformed, y, categorical_transformed = meta_train_data_transformed
         # TODO: somehow compute the expected output?
-        self.mf["LandmarkLDA"](self.X_transformed, self.y, self.logger)
-
-    def test_landmark_lda_multilabel(self):
-        X, y = self.get_multilabel()
-        mf = self.mf["LandmarkLDA"](X, y, self.logger)
-        self.assertTrue(np.isfinite(mf.value))
+        meta_features.metafeatures["SkewnessMin"](
+            X_transformed, y, logging.getLogger('Meta'), categorical_transformed)
 
-    def test_landmark_naive_bayes(self):
+    def test_skewness_max(self, meta_train_data_transformed):
+        X_transformed, y, categorical_transformed = meta_train_data_transformed
         # TODO: somehow compute the expected output?
-        self.mf["LandmarkNaiveBayes"](self.X_transformed, self.y, self.logger)
+        meta_features.metafeatures["SkewnessMax"](
+            X_transformed, y, logging.getLogger('Meta'), categorical_transformed)
 
-    def test_landmark_naive_bayes_multilabel(self):
-        X, y = self.get_multilabel()
-        mf = self.mf["LandmarkNaiveBayes"](X, y, self.logger)
-        self.assertTrue(np.isfinite(mf.value))
+    def test_skewness_mean(self, meta_train_data_transformed):
+        X_transformed, y, categorical_transformed = meta_train_data_transformed
+        # TODO: somehow compute the expected output?
+        meta_features.metafeatures["SkewnessMean"](
+            X_transformed, y, logging.getLogger('Meta'), categorical_transformed)
 
-    def test_landmark_decision_tree(self):
+    def test_skewness_std(self, meta_train_data_transformed):
+        X_transformed, y, categorical_transformed = meta_train_data_transformed
         # TODO: somehow compute the expected output?
-        self.mf["LandmarkDecisionTree"](self.X_transformed, self.y, self.logger)
+        meta_features.metafeatures["SkewnessSTD"](
```

Comment:
There was a check for the output of the meta-feature skewness std. Could you please re-add that?

Reply:
Sorry for the trouble, it was re-located a few lines below, [here](https://github.com/automl/auto-sklearn/blob/93e963c9db45803920f9524e3641a63c1f7ea64a/test/test_metalearning/pyMetaLearn/test_meta_features.py#L412). The reason why I moved it is that I grouped all test with transformed X.